extensions/wikia/Wall/WallMessage.class.php

```diff
@@ -271,11 +156,11 @@ public function setOrderId( $val = 1 ) {
 	}
 
 	/**
-	 * @return CommentsIndex
+	 * @return CommentsIndexEntry
 	 */
-	public function getCommentsIndex() {
+		public function getCommentsIndexEntry() {
 		if ( false === $this->commentsIndex ) { // false means we didn't call newFromId yet
-			$this->commentsIndex = CommentsIndex::newFromId( $this->getId() ); // note: can return null
+			$this->commentsIndex = CommentsIndex::singleton()->entryFromId( $this->getId() ); // note: can return null
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)